tmt/checks/avc.py

```diff
@@ -282,6 +282,18 @@ class AvcDenials(CheckPlugin[Check]):
         check:
           - name: avc
 
+    .. note::
+
+        To work correctly, check requires SELinux to be enabled on the
```

Comment:
```suggestion
        To work correctly, the check requires SELinux to be enabled on the
```
or `this` or `avc check` or..?